sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -31,6 +31,86 @@ def create_endpoints(plugin):
     # stock tickers are 1-5 capital characters, with a dot allowed. To keep
     # this from triggering with random text we require a leading '$'
     plugin.on_message("s\$[A-Z\.]{1,5}", stock_quote, wait=False)
+    plugin.on_message("c\$[A-Z\.]{1,5}", crypto_quote, wait=False)
+
+async def crypto_quote(message, app):
+    match = STOCK_REGEX.search(message.get("text", ""))
+    if not match:
+        return
+
+    symbol = match.group("symbol")
+    LOG.debug("Fetching crypto quotes for symbol %s", symbol)
+
+    response = message.response()
+    try:
+        stocks = app["plugins"]["stocks"]
+        crypto = (await stocks.crypto())
+        quote = ''
+        for c in crypto:
+            if c["symbol"] == f"{symbol}USDT":
+                quote = c
```

Comment:
This is a bit weird. Is there no way to get only one quote from the API and not all crypto stocks ?

Either way I would add a `break` here to short-circuit the loop once the quote is found

Reply:
Unfortunately the functionality with IEX is to return a JSON file of all their crypto assets. This is a somewhat limited list since they only support the more popular and stable cryptos and not all of the altcoins. Good call on the `break`!